sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -2504,28 +2499,28 @@
           "type": "integer"
         },
         "deviceEtag": {
-          "description": "Device's ETag",
+          "description": "TODO - which operation does this affect?",
           "type": "string"
         },
         "status": {
-          "description": "Gets the corresponding Device's Status.",
+          "description": "Gets the device's status.",
           "enum": [
             "enabled",
             "disabled"
           ],
           "type": "string"
         },
         "statusReason": {
-          "description": "Reason, if any, for the corresponding Device to be in specified Status",
+          "description": "Reason, if any, for the device to be in specified status.",
           "type": "string"
         },
         "statusUpdateTime": {
           "format": "date-time",
-          "description": "Time when the corresponding Device's Status was last updated",
+          "description": "Time when the corresponding device's status was last updated.",
```

Comment:
"Last device status update time"?

Reply:
This sounds like we're referring to a "last" device; I'll leave it as is, unless you have some other concerns.

Reply:
We use this language in other places too. That is more personal choice at this point as none of us are content writers. Keep what you prefer.